src/components/home.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { DeleteApiAction, GetApiAction } from "../redux/action/action";
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const responseData = useSelector((state) => state.reducer.details); //reducer variable name in rootReducer or details
+
+  const handleDelete = (id) => {
+    dispatch(DeleteApiAction(id));
+    window.location["reload"]();
```

Comment:
Why are you using window.location directly ? .Your component should update the state.